src/wazuh_modules/wm_osquery_monitor.c

```diff
@@ -268,7 +268,7 @@ void *Execute_Osquery(wm_osquery_monitor_t *osquery)
     }
 
     mdebug1("Launching '%s' with config file '%s'", osqueryd_path, osquery->config_path);
-    snprintf(config_path, sizeof(config_path), "--config_path=%s", osquery->config_path);
+    snprintf(config_path, sizeof(config_path), "--config_path=\"%s\"", osquery->config_path);
```

Comment:
Please test this change in the Linux version of the module as well to ensure it doesn't affect it.

Reply:
Done!